pkg/common/patterns.go

```diff
@@ -58,14 +58,14 @@ func (r RegexState) Matches(data []byte) []string {
 
 // UsernameRegexCheck constructs an username usernameRegex pattern from a given pattern of excluded characters.
 func UsernameRegexCheck(pattern string) RegexState {
-	raw := fmt.Sprintf(`(?im)(?:user|usr)\S{0,40}?[:=\s]{1,3}[ '"=]{0,1}([^:%+v]{4,40})\b`, pattern)
+	raw := fmt.Sprintf(`(?im)(?:user|usr)\S{0,40}?[:=\s]{1,3}[ '"=]{0,1}([^:%+v'"\s]{4,40})`, pattern)
```

Comment:
@dylanTruffle to clarify, the reason for removing `/b` in username was to be able to detect it in the middle of URL, right?